packages/gatsby-theme-carbon/src/components/LeftNav/LeftNavItem.js

```diff
@@ -16,9 +17,18 @@ export const SERVICE_WORKER_UPDATE_FOUND = 'GTC-ServiceWorkerUpdateFound';
 
 const LeftNavItem = (props) => {
   const { items, category, hasDivider } = props;
-  const { toggleNavState } = useContext(NavContext);
+  const { toggleNavState, leftNavIsOpen } = useContext(NavContext);
   const { isServiceWorkerEnabled } = useMetadata();
   const isOnline = useNetworkState();
+  const isLgWindow = window.matchMedia(`(min-width: ${breakpoints.lg.width} )`);
+
+  useEffect(() => {
+    if (isLgWindow) {
+      toggleNavState('leftNavIsOpen', 'open');
+    } else {
+      toggleNavState('leftNavIsOpen', 'close');
+    }
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    if (typeof window !== 'undefined') {
      const isLgWindow = window.matchMedia(
        `(min-width: ${breakpoints.lg.width})`
      ).matches;

      if (isLgWindow) {
        toggleNavState('leftNavIsOpen', 'open');
      } else {
        toggleNavState('leftNavIsOpen', 'close');
      }
    }
  }, [toggleNavState]);
```

window isn't available during build, thats why the deployment is failing. I think this should fix things